colcon_cmake/task/cmake/build.py

```diff
@@ -313,16 +314,30 @@ def _get_make_arguments(self):
 
         if CMAKE_EXECUTABLE is None:
             raise RuntimeError("Could not find 'cmake' executable")
-        cmd = [
-            CMAKE_EXECUTABLE, '--build', args.build_base,
-            '--target', 'install']
+        cmake_ver = await get_cmake_version()
```

Comment:
Currently the return value is not being checked for a failure.

Reply:
Do you mean failure to parse - i.e, checking for a version number result which indicates failure -  or for failure of the await directive?

Reply:
Your custom function might return `(0, 0, 0, 'failed')`.

Reply:
The return values have changed. This is no longer relevant.

Reply:
While the way the version is parsed has changed the fact that the parsing can fail for an invalid string hasn't. And that case should be explicitly handled.